src/navigators/AppStack.js

```diff
@@ -15,6 +19,17 @@ const AppStack = () => (
       component={MainScreen}
       options={{ title: strings.MAIN_SCREEN.title }}
     />
+    <Stack.Screen
+      name={CHAT_SCREEN}
+      component={ChatScreen}
+      options={({ route }) => ({
+        title: route.params.matchedUser.fullName,
+        headerTintColor: BLACK,
+        headerBackTitle: ' ',
+        // eslint-disable-next-line react/no-multi-comp
+        headerBackImage: () => <Image source={backIcon} style={{ marginLeft: 18 }} />,
```

Comment:
will you be using the back image button for another screen?  If so, you could have a GoBack component or something like that.  Also this will be a TouchableOpacity, right? You could have a comment like // TODO: go to the previous screen